### PR TITLE
{bio}[intel/2020a] velocyto v0.17.17 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
node128.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/67585f31d866260eb7c016016165fa51 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a532399e24c2fc048ef5088e51b49780 for a full test report.

**Comment by deniskristak** [comment]
https://github.com/hpcugent/eb_inuits/issues/165

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'velocyto.py'
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:3]: I think using `velocyto` as name makes more sense, the Python aspect is already reflected in the `versionsuffix`; see also https://pypi.org/project/velocyto/

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('numpy-groupies', '0.9.13', {
+        'modulename': 'numpy_groupies',
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:29]: This should not be required, the auto-guessed modulename should be correct?

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        'source_tmpl': 'numpy_groupies-%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:30]: You can remove this too if you use `numpy_groupies` as name instead two lines higher

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        'checksums': ['7b17b291322353f07c51598512d077e3731da0a048cfa8f738f3460d1ef0658d'],
+    }),
+    ('loompy', '3.0.6', {
+        'checksums': ['58e9763b8ab1af2a4a0e3805d120458b5184fd2b0f3031657ecce33c63ca4c46'],
+    }),
+    (name, version, {
+        'modulename': 'velocyto',
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:37]: No longer needed if you use `velocyto` as software name

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/velocyto-team/velocyto.py/archive'],
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:39]: Why not use `PyPI` here too, see https://pypi.org/project/velocyto/#files

File: easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/velocyto'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    'velocyto'
```
Comment (boegel) [inline easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb:52]: style nitpicking: add trailing comma:
```python
sanity_check_commands = [
    'velocyto',
]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
@deniskristak thanks for addressing the review. I have also removed the easyconfig with the old name from this PR. In the future, if you ever need to delete easyconfigs from PRs using `eb`, you can do so by prepending `:` to the name of the easyconfig. For instance, in this case I did

```
eb --update-pr 11744 --pr-commit-msg "remove velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb" :velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node378.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/471b11847079a8bea11009e54a7d1363 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node128.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/77c857687ba2a785740d40feca2e8adc for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @deniskristak !

**Pull request merged**